### DISCUSSION TITLE
Request received! We found 4 bindable values

### DISCUSSION THREAD

**Discussion by dannyspanish** [Budibase/budibase#4735] [Help]
Hi,

I'm super new to Budibase (it's awesome BTW) but I'm having trouble understanding how the code parts work. Specifically, I'd like to use the webhook to grab submitted form data from a website. Using the automate function, I'm able to send the data to Budibase. It all looks connected because it says 'Request received! We found 4 bindable values'. How do I access those values? how do I know what was sent? I'm using the cloud based version, if that helps.

I'd like to add a new row to an existing table (budibase), I'd like to add the email, first name, last name and phone number data that is captured from my form.

Any help would be really appreciated. 

Cheers, Danny,

**Comment by mjashanks** [comment]
Hey Danny. Welcome! Apologies for the late reply.

I setup a webhook, and set the following JSON

```json
{
    "name": "michael",
    "date": "2022-01-01T00:00:00.000Z"
}
```

Then, I added and email step. When I click the binding button (lightning bolt), beside the "Send To" field, I can use the 2 values that I sent with the JSON. YOu can use these values in any automation step - including `Save Row`

![image](https://user-images.githubusercontent.com/3524181/156545239-8e170f20-69d8-44c6-acc3-4287cf1b4a6a.png)

**Reply by dannyspanish** [reply]
Hi Michael, thanks for getting back to me. It's handy to see how the triggers look on other web hooks. I think for me to get a better understanding I really need to see what Budibase has received or what my website has sent. Is it possible to view the received webhook data in a more raw form in Budibase? I suspect that all of my field data is stored in one string but I don't know what I'm looking for exactly. {{ trigger.fields }} appears to be blank.

 
<img width="547" alt="Screenshot 2022-03-04 at 08 29 33" src="https://user-images.githubusercontent.com/34600678/156727673-511779e5-b161-45c4-b445-d1cdff0e1cd1.png">
